src/main/java/games/strategy/engine/lobby/server/db/UserDao.java

```diff
@@ -9,10 +9,16 @@
   boolean doesUserExist(String userName);
 
   void updateUser(DBUser user, HashedPassword password);
+  
+  void updateUser(DBUser user, String password);
```

Comment:
Yes this is the intention.
Once DbUserController is being merged with UserController the HashedPassword methods are supposed to be removed along with the HashedPassword object which is essentially just a wrapper for a string. (If we no longer require the client to send hashed passwords, we don't need to check if they are valid hashes)